.github/workflows/windows.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [windows]
-        python-version: ['3.10', '3.11']
+        python-version: ['3.7', '3.8']
```

Comment:
we probably shouldn't use 3.7  since that will be disappearing after release.

Reply:
I just wanted to see wether this fixes the error or not. Now I reverted it to the state in master.  I will look at how to solve this in a different branch.